src/components/gds-card/_index.scss

```diff
@@ -0,0 +1,13 @@
+@mixin card {
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  text-align: center;
+  border-radius: 8px;
+  background-color: white;
+  box-shadow: 5px 5px 15px #64646426, -5px -5px 15px #ffffffcc;
+  overflow: hidden;
+  min-width: 356px;
```

Comment:
Might be nice to extract magical numbers as variables (`$gds-card-width`, `$gds-card-min-width`) to the top of the file so they can be easily seen and changed? I guess they might occasionally even be needed by complex components that use this one?

Reply:
Yeah, that's true. I already do it with the logo grid. I'll add the variable.